pages-activities/activities-learning.html

```diff
@@ -174,4 +174,23 @@ <h4>
             </div>
         </div>
     </div>
+    <div class="col-lg-4 col-md-6 d-flex align-items-stretch">
+        <div class="row">
+            <div class="card">
+                <h4>
+                    <span class="fas fa-frog"></span>
+                    <a href="https://flexboxfroggy.com/" target="_blank">FlexBox Froggy</a>
+                    <span class="badge badge-warning">Easy</span>
+                </h4>
+                <p>
+                    Flexbox Froggy is a game for learning CSS flexbox.<br/>
+                    It takes inspiration from the classic arcade game 
+                    <a href="https://en.wikipedia.org/wiki/Frogger">Frogger</a>, 
+                    and web literacy games like the brilliant 
+                    <a href="https://flukeout.github.io/" style="white-space: nowrap;">CSS Diner</a> 
+                    and <a href="https://eraseallkittens.com/">Erase All Kittens</a>, 
```

Comment:
We already have CSS Diner listed, but would you mind adding Erase All Kittens as well?

Reply:
Done too :)